flapper-news/public/javascripts/posts/posts.factory.js

```diff
@@ -1,14 +1,14 @@
 // needed to inject $http
-app.factory('posts', ['$http', [function($http){
-	
+app.factory('posts', ['$http', function($http){
```

Comment:
This was the hardest thing to track down.  At first glance all looks correct.  The way I ended up finding this was by putting `console`'s at the top of each file, inside the `resolve` of the route, inside the `getAll` method.  Nothing inside the `posts` factory was being called which led me to believe that it was the culprit.  

Just by luck, I scrolled to the bottom of the file and saw `}]]);` which looked really out of place, I scrolled to the top and sure enough there was an extra square bracket before the `function`.  That caused all sorts of crazy.